src/site/content/en/shows/show/index.11tydata.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module.exports = {
+  pagination: {
+    before: (shows) => shows.reduce((a, c) => [...a, ...c.elements], []),
```

Comment:
Can you leave a comment here? `a` and `c` do not good variable names make.

Reply:
Renamed them to `all` and `current`.